sdk/communication/Azure.Communication.Identity/CHANGELOG.md

```diff
@@ -6,9 +6,12 @@
 - Added CommunicationIdentityClient (originally was part of the Azure.Communication.Administration package).
 - Added support to create CommunicationIdentityClient with TokenCredential.
 - Added support to create CommunicationIdentityClient with AzureKeyCredential.
+- Added ability to create a user and issue token for it at the same time.
 
 ### Breaking
 - CommunicationTokenScope.Pstn is removed.
+- Revoking tokens now revoke all the currently issued tokens instead of revoking tokens issued prior to a given time.
```

Comment:
Suggest changing this to CommunicationIdentityClient.RevokeTokens to match line 14.